packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -101,28 +111,312 @@ export class JobService {
+          };
+
+        case JobCaptchaShapeType.CATEGORAZATION:
+          return {
+              ...commonManifestProperties,
+              request_type: JobCaptchaRequestType.IMAGE_LABEL_MULTIPLE_CHOICE,
+              groundtruth_uri: jobDto.annotations.groundTruths,
+              restricted_audience: {
+                sitekey: [
+                  {
+                    [this.configService.get<number>(ConfigNames.HCAPTCHA_SITE_KEY)!]: {
+                      score: 1
+                    }
+                  }
+                ]
+              },// this.buildHCaptchaRestrictedAudience(jobDto.advanced),
+              requester_restricted_answer_set: this.buildHCaptchaRestrictedAnswerSet(groundTruthsData)
+          };
+
+        case JobCaptchaShapeType.POLYGON:
+          if (!jobDto.annotations.label) {
+            this.logger.log(ErrorJob.JobParamsValidationFailed, JobService.name);
+            throw new BadRequestException(ErrorJob.JobParamsValidationFailed);
+          }
+          
+          const polygonManifest = {
+              ...commonManifestProperties,
+              request_type: JobCaptchaRequestType.IMAGE_LABEL_AREA_SELECT,
+              request_config: {
+                  shape_type: JobCaptchaShapeType.POLYGON,
+                  min_shapes_per_image: HCAPTCHA_MIN_SHAPES_PER_IMAGE,
+                  max_shapes_per_image: HCAPTCHA_MAX_SHAPES_PER_IMAGE,
+                  min_points: 4,
+                  max_points: 4,
+                  minimum_selection_area_per_shape: HCAPTCHA_MINIMUM_SELECTION_AREA_PER_SHAPE,
+              },
+              groundtruth_uri: jobDto.annotations.groundTruths,
+              restricted_audience: {
+                sitekey: [
+                  {
+                    [this.configService.get<number>(ConfigNames.HCAPTCHA_SITE_KEY)!]: {
+                      score: 1
+                    }
+                  }
+                ]
+              },// this.buildHCaptchaRestrictedAudience(jobDto.advanced),
+              requester_restricted_answer_set: { [jobDto.annotations.label!]: { en: jobDto.annotations.label } },
+              requester_question_example: jobDto.annotations.exampleImages || [],
+          };
+
+          return polygonManifest;
+
+        case JobCaptchaShapeType.POINT:
+          if (!jobDto.annotations.label) {
+            this.logger.log(ErrorJob.JobParamsValidationFailed, JobService.name);
+            throw new BadRequestException(ErrorJob.JobParamsValidationFailed);
+          }
+
+          const pointManifest = {
+              ...commonManifestProperties,
+              request_type: JobCaptchaRequestType.IMAGE_LABEL_AREA_SELECT,
+              request_config: {
+                  shape_type: jobType,
+                  min_shapes_per_image: HCAPTCHA_MIN_SHAPES_PER_IMAGE,
+                  max_shapes_per_image: HCAPTCHA_MAX_SHAPES_PER_IMAGE,
+                  min_points: 1,
+                  max_points: 8,
```

Comment:
why are those values constant?

Reply:
These are the most preferred object settings for hCaptcha. They have been removed to keep the interface simple. Also, their implementation required additional time, since all these values must be correctly validated depending on the selected type.

Reply:
we should maybe use constants here then